Sources/SwiftGRPC/Core/Channel.swift

```diff
@@ -40,12 +40,15 @@ public class Channel {
   ///
   /// - Parameter address: the address of the server to be called
   /// - Parameter secure: if true, use TLS
-  public init(address: String, secure: Bool = true) {
+  /// - Parameter args: list of arguments
```

Comment:
A bit non-descriptive, how about e.g. "list of channel configuration options"? Up to you. (Also consider renaming the parameter.)